text/0118-search-api.md

```diff
@@ -282,37 +282,21 @@ Cropped query terms are counted as a word regarding `cropLength`.
 
 Sending more query terms than the `cropLength` value has no impact. The cropped part will contain the `cropLength` number.
 
-###### 3.1.12.1.3. Length Reassignment to the left.
+###### 3.1.12.1.2. Keeping a phrase context
 
-Given an attribute defined in `attributesToCrop` containing:
-
-`"In his ravenous hatred he found no peace, and with boiling blood he scoured the umbral plains, seeking vengence afgainst the dark lords who had robbed him."`
-
-With `croplength` defined as `5` and `q` defined as `had robbed`, the cropped value will be:
+Meilisearch keep words that are farther from the matching words but are in the same phrase, than words that are nearer but separated by a dot.
```

Comment:
```suggestion
When determining where to crop, Meilisearch searches for matches that are maybe further from each other but that are present in the same sentence instead of matches that are close but separated by a dot (end of sentence).
```

Reply:
Or is it just what the example below would suggest, that if it makes no difference to criteria such as number of unique matches, Meilisearch will try to start or end the crop at the beginning or end of the sentence?

I think the example could be improved if you changed the sentence to something like

`"Eventually, Natalie and her split up. Split The World is a book written by Emily Henry. The world never fully recognized her talent."`

I think that would provide a clearer view of the edge cases

Reply:
In this case, do you have an idea on how to rephrase it @dichotommy

Reply:
I prefer the first one!